zds/utils/forms.py

```diff
@@ -108,8 +109,10 @@ def validate_string_list(self, string_list):
         :return: ``True`` if ``v`` is fully valid, ``False`` if at least one error appears. See ``self.errors``
         to get all internationalized error.
         """
+        string_list = list(filter(lambda s: s.strip(), string_list))
```

Comment:
plutôt `s.strip() != ""`. J'aime pas trop qu'on se base sur le fait qu'un string vide est false-y.

Reply:
why not même si je pense le contraire de toi.